recipe/meta.yaml

```
@@ -7,10 +7,10 @@ package:
 source:
   fn: sunpy-{{version}}.tar.gz
   url: https://pypi.io/packages/source/s/sunpy/sunpy-{{ version }}.tar.gz
-  sha256: 1e7bb32d0851c1e01f202a80f1c35e16618b6c21325ff1fb28cfe138e658c42c
+  sha256: 08b70cbf1c7f641eacc06b23bc54873dc2d397e0a97991e806801ed1e0177b2b
 
 build:
-  number: 1000
+  number: 0
   script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
```

you should probably try: `

```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
```